packages/redux-query/src/middleware/query.js

```diff
@@ -33,51 +60,99 @@ const defaultConfig = {
     statusCodes.SERVICE_UNAVAILABLE,
     statusCodes.GATEWAY_TIMEOUT,
   ],
-  getQueryKey,
 };
 
-const getPendingQueries = queries => {
-  return pickBy(queries, query => query.isPending);
+const getPendingQueries = (queries: QueriesState): QueriesState => {
+  const pendingQueries = {};
+
+  for (const queryKey in queries) {
+    if (queries.hasOwnProperty(queryKey)) {
+      const query = queries[queryKey];
+
+      if (query.isPending) {
+        pendingQueries[queryKey] = query;
+      }
+    }
+  }
+
+  return pendingQueries;
+};
+
+const pick = (source: { [key: string]: any }, keys: Array<string>): { [key: string]: any } => {
```

Comment:
The general strategy in this project is to  use lodash functions (citing Ryan). Why is this custom implementation here? I believe there should be a consistency - if Lodash is a preferrable lib - then it should be used, and if custom implementations are required - why not to get rid of Lodash alltogether (which I believe is a good idea, as it would save about 30% of lib size).

Reply:
As I've migrated everything to flow I have reversed my decision on that. Getting flow to work well with other libraries is a pain (unless it's something like idx that just works automatically).

My thinking on this has also evolved quite a bit. I think your original point about bundle size is valid, and generally shedding dependencies, if not really necessary, is a good thing.